### DISCUSSION TITLE
Man-made objects in the forest

### DISCUSSION THREAD

**Discussion by aeszter** [streetcomplete/StreetComplete#5201] [Ideas]
Are there any ideas around for how to deal with man-made objects in the forest (or other natural surroundings)? Around here, the things that come to mind are benches/picnic tables, trash cans, hunting stands, and guide posts. In my experience, these objects are somewhat volatile (at least the first three mentioned). SC already deals quite well with disappearing objects via the "is this still here" quest (but not for hunting stands) -- but I don't see a good way of adding them if missing. Right now, I switch to Vespucci or (when bendwidth is a problem) OsmAnd. Maybe an overlay would be an option here?

**Comment by Helium314** [comment]
Hunting stands could possibily be added to the planned street furniture overlay #4912 (though hunting stands are usually nowhere near streets)

**Reply by aeszter** [reply]
Thanks for the pointer, that's almost exactly what I was looking for. 
And yes, it would be nice if hunting stands could be added to that overlay (and to the existence quest as well).
